ibis/backends/duckdb/__init__.py

```diff
@@ -506,6 +507,39 @@ def read_sqlite(self, path: str | Path, table_name: str | None = None) -> ir.Tab
 
         return self.table(table_name)
 
+    def attach_sqlite(
+        self, path: str | Path, overwrite: bool = False, all_varchar: bool = False
+    ) -> None:
+        """Attach a SQLite database to the current DuckDB session.
+
+        Parameters
+        ----------
+        path
+            The path to the SQLite database.
+        overwrite
+            Allow re-scan of SQLite database of the same name.
```

Comment:
Can you expand this comment slightly, it's not 100% clear to me what this option does.

Reply:
Done